cbor_event/src/se.rs

```diff
@@ -436,6 +441,11 @@ impl<W: Write+Sized> Serializer<W> {
         self.write_type(Type::Tag, tag)
     }
 
+    /// write a tag that indicates that the following list is a finite set
```

Comment:
According to a comment in `cardano-sl`, it will be standardized soon:
```
-- We stitch a `258` in from of a (Hash)Set, so that tools which
-- programmatically check for canonicity can recognise it from a normal
-- array. Why 258? This will be formalised pretty soon, but IANA allocated
-- 256...18446744073709551615 to "First come, first served":
-- https://www.iana.org/assignments/cbor-tags/cbor-tags.xhtml Currently `258` is
-- the first unassigned tag and as it requires 2 bytes to be encoded, it sounds
-- like the best fit.
```
I've added a reference to the IANA list.